pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/signed-exchange.md

```diff
@@ -108,111 +107,69 @@ Ea8/B6hPatJ0ES8q/HO3X8IVQwVs1n3aAr0im0/T+Xc=
+### Testing
 
-Test that your staging site returns the content with the added MIME type `application/signed-exchange` when provided with the correct request headers. In the below example, replace `staging.example.com` with your staging server.
+Verify that your staging site returns the content with the added MIME type `application/signed-exchange` when provided with the correct request headers. In the below example, replace `staging.example.com` with your staging server.
 
-
-```html
-$ curl -si -H 'amp-cache-transform: google' -H 'accept: application/signed-exchange;v=b3;q=0.9,*/*;q=0.8' https://staging.example.com/ | less
+```sh
+$ curl -si -H 'amp-cache-transform: google' -H 'accept: application/signed-exchange;v=b3;q=0.9,*/*;q=0.8' https://staging.example.com/
 ```
 
 This command should return the following:
 
-```html
+```txt
 content-type: application/signed-exchange;v=b3
 ```
 
 [tip type="important"]
 The `v=b3` version string is the current version. This version will change. 
 [/tip]
 
-The response will be a binary encoding of your AMP page.
-
-You can also test with the accept header Chrome sends on navigations:
+The bulk of the response should be your AMP page (in plaintext) together with some binary headers and footers.
 
+You can also test with the more complicated accept header sent by Chrome:
 
-```html
-$ curl -si -H 'amp-cache-transform: google' -H 'accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3' https://staging.example.com/ | less
+```sh
+$ curl -si -H 'amp-cache-transform: google' -H 'accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3' https://staging.example.com/
 ```
 
+The [`dump-signedexchange` tool](https://github.com/WICG/webpackage/blob/master/go/signedexchange/README.md#installation) can be used to inspect the response:
 
-Use the [`dump-signedexchange` tool](https://github.com/WICG/webpackage/blob/master/go/signedexchange/README.md#installation) to read the Signed Exchange output. 
-
-
-```html
+```sh
 $ curl -s --output - -H 'amp-cache-transform: google' -H 'accept: application/signed-exchange;v=b3;q=0.9,*/*;q=0.8' https://staging.example.com/ > example.sxg
 $ dump-signedexchange -i example.sxg
 format version: 1b3
 ```
 
+(Note that `-verify` switch will not work at this point because the required certificates are not on the `https://example.com/` server.)
 
-The `-verify` switch will not work at this point because the required certificates are not on the `https://example.com/` server. 
-
-
-## Publish content and direct traffic 
+## Deploy packager to production
 
-Ensure your signed documents have the correct headers needed to find and filter content. These headers can be found in the [Productionizing section of the `amppackager` README](https://github.com/ampproject/amppackager/blob/master/README.md#productionizing). 
+### Test with command-line tools
```

Comment:
Seems to be missing a step. Went straight from "test staging" to "test production" without saying to deploy to production (other than in the h2).

Reply:
Added: "Adjust the staging deployment steps above as appropriate for your production environment."